.github/ISSUE_TEMPLATE/vagrant-feature-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Vagrant Feature request
+about: Suggest an idea or enhancement for Vagrant
+title: 'Enhancement Request: Your description here'
+labels: ''
```

Comment:
I think it would make sense to automatically add `enhancement` labels to issues generated with this template.